articles/compliance/gdpr/features-aiding-compliance/data-portability.md

```diff
@@ -26,9 +26,4 @@ You can export a user's full profile using our Management API. The response will
 - [Search for a user using their email address](/users/search#users-by-email)
```

Comment:
Thinking that for GDPR purposes, might be better to only list GET by ID and User Export. User Search shouldn't be used (and won't be available) on many scenarios.